app/electron/main.ts

```diff
@@ -488,11 +560,18 @@ function startServer(flags: string[] = []): ChildProcessWithoutNullStreams {
   serverArgs = serverArgs.concat(flags);
   console.log('arguments passed to backend server', serverArgs);
 
-  // We run detached but not in shell, otherwise it's hard to make sure the
-  // server process gets killed. When changing these options, please make sure
-  // to test quitting the app in the different platforms and making sure the
-  // server process has been correctly quit.
-  const options = { detached: true };
+  const extendedEnv = getShellEnv();
+  const options = {
+    detached: true,
+    env: {
+      ...extendedEnv,
+      HEADLAMP_CONFIG_ENABLE_HELM: 'true',
```

Comment:
Can you please explain why these environment variables are added?

They are supposed to be user supplied, so they shouldn't be set here.

Reply:
Since we ship the app store plugin in the app we already set it up [here](https://github.com/headlamp-k8s/headlamp/blob/c26c9137f21312a925d85a63f41808d573cb4a0f/app/electron/main.ts#L480). I was adding it for debugging the `envfiles.txt`. I will remove this.